src/signals/IncidentMap/components/FilterPanel/utils.test.ts

```diff
@@ -34,3 +38,19 @@ describe('getFilterCategoriesWithIcon', () => {
     )
   })
 })
+
+describe('showSubCategoryFilter', () => {
+  it('should return false on default', () => {
```

Comment:
might wanne add a configuration mock reset here, when adding a new test, you don't want configuration.map.optionsIncidentMap.hasSubfiltersEnabled be set is some other it block.

Reply:
good one!